tests/test_blocks/test_step_impl/test_ek1_projmatfree_d_nu.py

```diff
@@ -0,0 +1,103 @@
+"""Tests for traditional EK1 implementations."""
+
+from functools import partial
+
+import jax
+import jax.numpy as jnp
+import pytest
+
+from tornadox.blocks.sde import ibm
+from tornadox.blocks.step_impl.first_order import ek1_projmatfree_d_nu
```

Comment:
* I would've preferred `tornadox.building_blocks.step_implementation...` but it leads to way too long import statements. 
* ek1_projmatfree should be self-explanatory. The `d_nu` bit refers to the ordering of ode-dimension-vs-number-of-derivatives

Reply:
if that's the only ordering, should it really be in the filename? It's in the docstring after all

Reply:
I prefer to convey this info in the function name than in the docstring only.  At least, when writing code for myself...(I rarely read docstrings of functions I wrote myself. In fact, I rarely write docstrings for functions I write for myself... :smile:) But I see that this is a bit controversial.

I guess this also depends on the answer to your high-level comment regarding the deep nesting. So I'd say we discuss this first, and then finish this one here?